### ISSUE TITLE
Add support for proxy

### ISSUE THREAD

**Issue by ottuzzi** [cdhowie/Bitcoin-mining-proxy#39]
Hi,

   is it possible to add support for a proxy?
I'm behind a squid proxy and I cannot access internet directly: I'm not a PHP expert but it looks like you need to change function place_json_call() in common.inc.php.

Thanks
Bye
Piero

**Comment by cdhowie** [comment]
Yes, this can be done.  Note, however, that some proxies might not forward the relevant X-\* headers, so some features may not work.  Additionally, proxies might limit the request duration, so long-polling might not work at all.  These restrictions will vary depending on the proxy software/configuration.
